src/mlpack/methods/adaboost/adaboost_regressor_impl.hpp

```diff
@@ -0,0 +1,387 @@
+/**
+ * @file methods/adaboost/adaboost_regressor_impl.hpp
+ * @author Dinesh Kumar
+ *
+ * Implementation of Adaboost regressor class.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+
+#ifndef MLPACK_METHODS_ADABOOST_ADABOOST_REGRESSOR_IMPL_HPP
+#define MLPACK_METHODS_ADABOOST_ADABOOST_REGRESSOR_IMPL_HPP
+
+// In case it hasn't been included yet.
+#include "adaboost_regressor.hpp"
+
+namespace mlpack{
+
+// Construct, don't train
```

Comment:
From my point of view, this kind of comment is useless, to be honest, I would just remove it, no need for it at all